instrumentation/jaxrs/jaxrs-1.0/src/test/groovy/JaxRsAnnotations1InstrumentationTest.groovy

```diff
@@ -42,7 +41,6 @@ class JaxRsAnnotations1InstrumentationTest extends AgentTestRunner {
 
   def "span named '#name' from annotations on class when is not root span"() {
     setup:
-    def startingCacheSize = spanNames.size()
```

Comment:
These tests looked really had to do with the agent separation. We have other caches where we don't have this sort of test, so guessing it's not a huge regression.